api/src/main/java/io/lumeer/api/model/LinkType.java

```diff
@@ -19,24 +19,94 @@
 
 package io.lumeer.api.model;
 
+import io.lumeer.api.dto.JsonAttribute;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.Collections;
 import java.util.List;
 
-public interface LinkType {
+public class LinkType {
+
+   public static final String ID = "_id";
+   public static final String NAME = "name";
+   public static final String COLLECTION_IDS = "collectionIds";
+   public static final String ATTRIBUTES = "attributes";
+
+   private String id;
+   private String name;
+   private List<String> collectionIds;
+   private List<JsonAttribute> attributes;
+
+   @JsonCreator
+   public LinkType(@JsonProperty(NAME) final String name,
+         @JsonProperty(COLLECTION_IDS) final List<String> collectionIds,
+         @JsonProperty(ATTRIBUTES) final List<JsonAttribute> attributes) {
```

Comment:
We would probably want to map `id` field coming from the front-end as well, or not?

Reply:
Yes of course